autopush/tests/test_websocket.py

```diff
@@ -51,6 +51,20 @@
     "content-encoding": "aesgcm",
     "TTL": "60",
 }
+dummy_chid = str(uuid.uuid4())
+dummy_uaid = str(uuid.uuid4())
```

Comment:
..was thinking similarly but our other tests' dummy uuids are strs. but let's move that way anyway